action.yml

```diff
@@ -1,108 +1,41 @@
-name: Pros Build
-description: Automatically build pros projects
+name: 'ABUCKY0 PROS Build'
+description: 'Build a PROS Project'
+branding:
+  color: 'blue'
+  icon: 'box'
+  colorized: true
 
 inputs:
   repository:
     description: Target repository
     required: false
     default: ${{ github.repository }}
-  library-path:
+  library_path:
     required: false
     default: null
-    description: The path of the library inside of the `include` directory. Specifying a value for this input will cause the action to upload the built libary in template form as an artifact.
-
+    description: The path of the library inside of the `include` directory. Specifying a value for this input will cause the action to upload the built library in template form as an artifact.
+  after:
+    required: false
+    default: ${{ github.event.after }}
+    description: The commit hash to checkout
+  action:
+    required: false
+    default: ${{ github.event.action }}
+    description: The action that triggered the workflow
+  pr_num:
+    required: false
+    default: ${{ github.event.number }}
+    description: The PR number
```

Comment:
This isn't used and could probably be deleted (pr_num)

```yml
  pr_num:
    required: false
    default: ${{ github.event.number }}
    description: The PR number
````

I instead just directly use GITHUB_PR_NUM

Reply:
I removed this and a few other portions